torch_xla/torch_xla.py

```diff
@@ -100,20 +111,24 @@ def foo2(x):
         res = foo2(x)
   """
 
+  def _clear_pending_ops_before_compile():
```

Comment:
Thanks. Long-term, we should be able to remove this. If we assume eager execution outside of `compile`, then there should be no "pending ops"

Reply:
yea with eager there shouldn't be any pending ops.